pkg/api/universal/scheduler.go

```diff
@@ -19,15 +19,34 @@ import (
 
 	"google.golang.org/protobuf/types/known/emptypb"
 
+	apierrors "github.com/dapr/dapr/pkg/api/errors"
 	runtimev1pb "github.com/dapr/dapr/pkg/proto/runtime/v1"
 	schedulerv1pb "github.com/dapr/dapr/pkg/proto/scheduler/v1"
 )
 
 func (a *Universal) ScheduleJob(ctx context.Context, inReq *runtimev1pb.ScheduleJobRequest) (*emptypb.Empty, error) {
-	// validate job details, date, schedule, etc??
 	var err error
 
 	metadata := map[string]string{"app_id": a.AppID()}
+
+	if inReq.GetJob() == nil {
+		return &emptypb.Empty{}, apierrors.Empty("job", metadata, apierrors.CodePrefixScheduler+apierrors.PostFixEmpty)
+	}
+
+	if inReq.GetJob().GetName() == "" {
+		return &emptypb.Empty{}, apierrors.Empty("job name", metadata, apierrors.CodePrefixScheduler+apierrors.InFixJob+apierrors.InFixName+apierrors.PostFixEmpty)
+	}
+
+	if inReq.GetJob().GetSchedule() == "" {
+		return &emptypb.Empty{}, apierrors.Empty("job schedule", metadata, apierrors.CodePrefixScheduler+apierrors.InFixSchedule+apierrors.PostFixEmpty)
```

Comment:
Add integration tests for these error codes.

Reply:
working on that now